esp/mlrs-wifi-bridge/mlrs-wifi-bridge-boards.h

```diff
@@ -69,6 +69,15 @@ UARTs can be mapped to any pins, according to data sheet
 Mates cleanly with R9M inverted serial port pins
 ATTENTION: when the 5V pin is used, one MUST not also use the USB port, since they are connected internally!!
 Disconnect from application before programming via USB. Hold down central button (G9) when connecting to USB to program
+
+------------------------------
+M5Stack ATOM Lite
+------------------------------
+board: M5Stack ATOM Lite
```

Comment:
here too should be M5Stack-ATOM
sorry for missing in first round

Reply:
Fixed